.github/CONTRIBUTING.md

```diff
@@ -1,112 +1,53 @@
 
-To run the build process, in a terminal go to your WooCommerce installation and run:
+If you have questions about the process to contribute code or want to discuss details of your contribution, you can contact WooCommerce core developers on the #core channel in the [WooCommerce community Slack](https://woocommerce.com/community-slack/).
 
-```bash
-npm install
-composer install
-npm run build
-```
+## Getting started
 
-To get your development environment running we recommend [reading here](https://github.com/woocommerce/woocommerce/wiki/How-to-set-up-WooCommerce-development-environment).
+- [How to set up WooCommerce development environment](https://github.com/woocommerce/woocommerce/wiki/How-to-set-up-WooCommerce-development-environment)
+- [Git Flow](https://github.com/woocommerce/woocommerce/wiki/WooCommerce-Git-Flow)
+- [Minification of SCSS and JS](https://github.com/woocommerce/woocommerce/wiki/Minification-of-SCSS-and-JS)
+- [String localisation guidelines](https://github.com/woocommerce/woocommerce/wiki/String-localisation-guidelines)
+- [Running unit tests](https://github.com/woocommerce/woocommerce/blob/master/tests/README.md)
+- [Running e2e tests](https://github.com/woocommerce/woocommerce/wiki/End-to-end-Testing)
 
 ## Coding Guidelines and Development 🛠
 
-- **Ensure you stick to the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/)**
-- Run our build process described in the section above, it will install our pre-commit hook, code sniffs, dependencies, and more.
+- Ensure you stick to the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/)
+- Run our build process described in the document on [how to set up WooCommerce development environment](https://github.com/woocommerce/woocommerce/wiki/How-to-set-up-WooCommerce-development-environment), it will install our pre-commit hook, code sniffs, dependencies, and more.
+- Whenever possible please fix pre-existing code standards errors in the files that you change. It is ok to skip that for larger files or complex fixes.
 - Ensure you use LF line endings in your code editor. Use [EditorConfig](http://editorconfig.org/) if your editor supports it so that indentation, line endings and other settings are auto configured.
 - When committing, reference your issue number (#1234) and include a note about the fix.
 - Ensure that your code supports the minimum supported versions of PHP and WordPress; this is shown at the top of the `readme.txt` file.
 - Push the changes to your fork and submit a pull request on the master branch of the WooCommerce repository.
-
-Please avoid modifying the change-log directly or updating the .pot files. These will be updated by the WooCommerce team.
+- Make sure to write good and detailed commit messages (see [this post](https://chris.beams.io/posts/git-commit/) for more on this) and follow all the applicable sections of the pull request template.
```

Comment:
I wonder if we should link [WordPress commit message guidelines](https://make.wordpress.org/core/handbook/best-practices/commit-messages/) instead of Chris Beams post.

Reply:
I found Chris Beams post on a quick search, I don't mind changing it to something else. IMHO, we shouldn't change to the WordPress commit message guidelines as it includes some details that are specific to the WP workflow and don't apply to the WooCommerce workflow (for example, include the component or focus name in the commit message summary and instructions that apply only to Trac like using square brackets to reference changeset).